src/test/groovy/org/gradle/android/SimpleAndroidApp.groovy

```diff
@@ -58,7 +64,7 @@ class SimpleAndroidApp {
                     }
                     dependencies {
                         classpath ('com.android.tools.build:gradle:$androidVersion') { force = true }
-                        classpath "org.gradle.android:android-cache-fix-gradle-plugin:${Versions.PLUGIN_VERSION}"
```

Comment:
should we remove `Versions.PLUGIN_VERSION` or make it point to the system property?

Reply:
Well Versions is part of the production code.  `Versions.PLUGIN_VERSION` is only used by the test fixtures.  And it requires configuration cache gymnastics in order to read the system property safely.  Rather than introduce that, it seemed cleaner to just remove it from the production code and handle it in the test fixture as a test runtime specific system property.